simple_examples.py

```diff
@@ -102,6 +115,14 @@ def validate_auth(cls, value: str) -> str:
             raise AuthorizationError(value=value)
         return value
 
+@typechecked
+class AssessmentType(Enum):
```

Comment:
Its also come to my attention that `@typechecked` doesn't do anything for an Enum, (doesn't really make sense, its never called). We should remove it from this and `TextAlign`.